apps/condo/pages/news/[id]/index.tsx

```diff
@@ -106,7 +106,7 @@ const NewsItemCard: React.FC = () => {
         objs: newsItemScopes,
         loading: newsItemScopesLoading,
         error: newsItemScopesError,
-    } = NewsItemScope.useObjects({
+    } = NewsItemScope.useAllObjects({
```

Comment:
need to test it for big buildings

Reply:
what do you mean why would that be a problem?

Reply:
What about scopes created for two sections, each with 120 flats?